docs/190-paas-operations/configuration/030-advanced-network.md

```diff
@@ -155,6 +118,42 @@ If you have any questions, you can request CluedIn support by sending an email t
 
     In Azure, you should then see a new Load Balancer resource called `kubernetes-internal`, which will be used for ingress. The original `kubernetes` then simply becomes egress only.
 
+## Integration options
+With Kubernetes now configured to use the internal load balancer for ingress, you will need to decide how you want to access this front end load balancer from your internal network. The two options supported are `Virtual network peering` and `Private link service with private endpoint`.
+
+**Note**: If your subnets overlap in any way, the only option will be `Private link service with private endpoint`.
+
+### Virtual network peering
+To configure virtual network peering, Microsoft have a [document](https://learn.microsoft.com/en-us/azure/virtual-network/tutorial-connect-virtual-networks-portal) explaining how to achieve this easily.
+
+As all resources have already been created, you only need to follow the `Create virtual network peer` step in the above document.
+
+### Private link service with private endpoint
+In the event there is overlapping of subnets, this is not a problem as `private endpoints` also work here. This is slightly more complicated than virtual network peering and we'll offer a bit of guidance here.
+
+Microsoft have a [document](https://learn.microsoft.com/en-us/azure/private-link/create-private-link-service-portal) explaining how to achieve this. 
+
+As AKS maintains the load balancers, and the vnet is created at deployment time (or an existing one is used), you only need to follow the steps `Create a private link service` and `Create a private endpoint` in the above document. 
+
+For the `Private Link Service`, ensure that the `Load balancer` is set to `kubernetes-internal` and uses the frontend IP address from the private range. The Source NAT vnet and subnet should default correctly.
+Ensure that Source NAT subnet is set to No and it is best to assign a Static Private IP address.
+
+![adv-network-pls-01.png](../../assets/images/ama/howtos/adv-network-pls-01.png)
+
+For the `Private endpoint`, ensure you select the virtual network that will be used to access CluedIn along with the subnet. It is recommended to statically allocate an IP address here as this will be used as the entry point for CluedIn and will be required as part of your DNS at a later stage.
+
+![adv-network-pe-01.png](../../assets/images/ama/howtos/adv-network-pe-01.png)
+
+### Next steps
+The next step is to test access from the private network to CluedIn to ensure that traffic can reach the endpoint as expected.
+By default ICMP packets do not respond, but HTTPS (443/TCP) will. It may error due to certificate, but this can be corrected.
```

Comment:
```suggestion
By default, ICMP packets do not respond, but HTTPS (443/TCP) will. It may cause an error due to certificate, but this can be corrected.
```